### DISCUSSION TITLE
Please patch for 3.0.15

### DISCUSSION THREAD

**Discussion by st-jliu** [quictls/openssl#172] [Releases]
https://www.cve.org/CVERecord?id=CVE-2024-5535

Please patch 3.0.15, we are using that LTS branch.

**Comment by mhdawson** [comment]
Is there a plan for a release of 3.0.15? The Node.js project is waiting on this do to releases with an update OpenSSL - https://github.com/nodejs/nodejs-dependency-vuln-assessments/issues/192

**Reply by wbl** [reply]
We think we've already done it. Can you guys be more specific about why you think we haven't?

**Comment by wbl** [comment]
I think we found the typo in the tag and fixed. Can you please confirm?
